Bobby/function_level2/primeno.py

```diff
@@ -0,0 +1,13 @@
+# write a program to find whether the given no is prime or not
+def prime_number(n):
+    count = 0
+    for i in range(1,n+1):
+        if n%i == 0:
+            count = count + 1
+    if count ==2:
+        return True
```

Comment:
Please test with different inputs and find what is the problem with this code  @gopi018  @Rajkumar-V